File: C++/Iniciante/1020.cpp
#include <stdio.h>
#include <iostream>

int main() {

  int dias, mes, ano;
  std::cin >> dias;
  
  ano = dias / 365;
  dias %= 365;

  mes = dias / 30;
  dias %= 30;

  printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", ano, mes, dias);
    
}

File: C#/Iniciante/1020.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int valor = Convert.ToInt32(Console.ReadLine());

        int ano = valor / 365;
        valor %= 365;
        
        int meses = valor / 30;
        valor %= 30;
        
        int dias = valor % 30;
        
        Console.WriteLine($"{ano} ano(s)");
        Console.WriteLine($"{meses} mes(es)");
        Console.WriteLine($"{dias} dia(s)");

    }

}
